cmd/addvalidators.go

```diff
@@ -244,7 +244,7 @@ func writeDepositDatas(ctx context.Context, clusterDir string, numOps int, secre
 
 	var depositDatas []eth2p0.DepositData
 	for i, val := range vals {
-		depositMsg, err := deposit.NewMessage(eth2p0.BLSPubKey(val.PublicKey), val.WithdrawalAddress, deposit.MaxValidatorAmount)
+		depositMsg, err := deposit.NewMessage(eth2p0.BLSPubKey(val.PublicKey), val.WithdrawalAddress, deposit.MaxDepositAmount)
```

Comment:
Does this mean that when manually adding validators we can't specify partial deposits?

Reply:
The spec did not mention this. Shall we support this command as well?

Reply:
If we should support this and update manifest schema, then I will leave it for another PR, otherwise this one will be too big.

Reply:
@xenowits confirmed this has to be done later for other commands as well under a different ticket